selfdrive/car/chrysler/carstate.py

```diff
@@ -3,150 +3,242 @@
+      ("Steering_Wheel_Angle", "Steering_Column_Angle_Status"),  # Steering Angle
+      ("Steering_Rate", "Steering_Column_Angle_Status"),  # Steering rate
+      ("Steering_Column_Torque", "EPS_2"),  # EPS Driver applied torque
+      ("EPS_Motor_Torque", "EPS_2"),  # EPS Motor Torque output
+      ("Torque_Overlay_Status", "EPS_2"),
+      ("Traction_Button", "Center_Stack_1"),  # Traction Control Button
+      ("LKAS_Button", "Center_Stack_1"),  # LKAS Button
+      ("Turn_Signal_Status", "Steering_Column_Commands"),  # Blinker
+      ("High_Beam_Lever_Status", "Steering_Column_Commands"),  # High Beam Lever
+      ("ACC_Accel", "Cruise_Control_Buttons"),  # ACC Accel Button
+      ("ACC_Decel", "Cruise_Control_Buttons"),  # ACC Decel Button
+      ("ACC_Cancel", "Cruise_Control_Buttons"),  # ACC Cancel Button
+      ("ACC_Distance_Dec", "Cruise_Control_Buttons"),  # ACC Distance Decrement Button
+      ("ACC_Distance_Inc", "Cruise_Control_Buttons"),  # ACC Distance Increment Button
+      ("ACC_Resume", "Cruise_Control_Buttons"),  # ACC Resume Button
+      ("Cruise_OnOff", "Cruise_Control_Buttons"),  # Cruise On Off Button
+      ("ACC_OnOff", "Cruise_Control_Buttons"),  # ACC On Off Button
+      ("COUNTER", "Cruise_Control_Buttons"),  # ACC Counter Button
+      ("ACC_Distance_Inc", "Cruise_Control_Buttons"),  # ACC Distance Increase Button
+      ("Driver_Door_Ajar", "BCM_1"),  # driver Door
+      ("Passenger_Door_Ajar", "BCM_1"),  # Passenger Door
+      ("Left_Rear_Door_Ajar", "BCM_1"),  # Driver Rear Door
+      ("Right_Rear_Door_Ajar", "BCM_1"),  # Passenger Rear Door
+      ("Driver_Seatbelt_Status", "ORC_1"),  # Driver Sear Belt
+      ("COUNTER", "EPS_2"),  # EPS Counter
     ]
 
     checks = [
       # sig_address, frequency
-      ("BRAKE_2", 50),
-      ("EPS_STATUS", 100),
-      ("SPEED_1", 100),
-      ("WHEEL_SPEEDS", 50),
-      ("STEERING", 100),
-      ("ACC_2", 50),
-      ("GEAR", 50),
-      ("ACCEL_GAS_134", 50),
-      ("WHEEL_BUTTONS", 50),
-      ("DASHBOARD", 15),
-      ("STEERING_LEVERS", 10),
-      ("SEATBELT_STATUS", 2),
-      ("DOORS", 1),
-      ("TRACTION_BUTTON", 1),
+      ("Transmission_Status", 50),
+      ("ESP_1", 50),
+      ("ESP_4", 50),
+      ("ESP_6", 50),
+      ("ESP_8", 50),
+      ("ECM_5", 50),
+      ("Steering_Column_Angle_Status", 100),
+      ("EPS_2", 100),
+      ("Center_Stack_1", 1),
+      ("Steering_Column_Commands", 10),
+      ("Cruise_Control_Buttons", 50),
+      ("BCM_1", 1),
+      ("ORC_1", 1),
     ]
 
     if CP.enableBsm:
       signals += [
-        ("BLIND_SPOT_RIGHT", "BLIND_SPOT_WARNINGS"),
-        ("BLIND_SPOT_LEFT", "BLIND_SPOT_WARNINGS"),
+        ("Blind_Spot_Monitor_Left", "BSM_1"),
+        ("Blind_Spot_Monitor_Right", "BSM_1"),
+      ]
+      checks += [("BSM_1", 2)]
+
+    if CP.carFingerprint in (CAR.RAM_1500, ):
+      signals += [
+        ("LKAS_Button", "Center_Stack_2"),  # LKAS Button
+        ("DASM_FAULT", "EPS_3"),  # EPS Fault
```

Comment:
@realfast is there a signal equivalent to this on other Chryslers?

Reply:
@sshane i assume you are referring to the DASM_Fault error? If so, yes that same fault exists. Check out the Ram_2500 DBC.